packages/indexer-client/src/types/serverModelTypes.ts

```diff
@@ -191,6 +195,17 @@ export interface IndexerServerStakingV2Staker {
   pool_share: number;
 }
 
+export interface IndexerServerVrtxSupplySnapshot {
+  timestamp: string;
+  vrtx_oracle_price: string;
+  cumulative_incentives: string;
+  cumulative_lba: string;
+  cumulative_supply_ecosystem: string;
+  cumulative_supply_treasury: string;
+  cumulative_investors_supply: string;
```

Comment:
Should we get backend to change the names such that the placement of `supply` in the naming is consistent? There are two fields with supply as a prefix and two with supply as a postfix